packages/angular-cli/tasks/build-webpack.ts

```diff
@@ -55,6 +60,19 @@ export default <any>Task.extend({
         }
         resolve();
       });
-    });
+    })
+      .then(() => {
+        if (appConfig.universal === false || runTaskOptions.environment !== 'prod') {
+          return;
+        }
+
+        const fileName = path.resolve(outputDir, appConfig.index);
+        const data = fs.readFileSync(fileName, 'utf8').replace(
+          '<script src="http://localhost:35729/livereload.js?snipver=1"></script>',
```

Comment:
I tried to replace it with webpack in the webpack-build-prod config. but I did not get it to work. I used the string-replacement plugin. I'll try it again. I just want to remove the live-reload-script in production.